components/Facility/index.js

```diff
@@ -44,20 +45,29 @@ export default function Facility({ facility, setFacilities }) {
                   : "/icons/isBookmarked.png"
               }
               alt="isNotBookmarkedHeart"
-              width={30}
-              height={30}
+              width={40}
+              height={40}
             />
           </StyledButton>
-          <ListItem>Zielgruppe: {target}</ListItem>
-          <ListItem>Zugangsvoraussetzungen: {requirements}</ListItem>
           <ListItem>
-            Adresse:{" "}
+            <b>Kategorie: </b> {facility.category}
```

Comment:
maybe it's better to style with <span> and style components like this:
`<styledSpan>Kategorie:</styledSpan>
const styledSpan = styled.span`
font-weight: bold;
`